CMake/ctkMacroCompilePythonScript.cmake

```diff
@@ -21,6 +21,9 @@ include(${CTK_CMAKE_DIR}/ctkMacroParseArguments.cmake)
 
 set(CTK_PYTHON_COMPILE_FILE_SCRIPT_DIR "${CMAKE_BINARY_DIR}/CMakeFiles")
 
+# For default global behavior, define CopySlicerPythonScriptFiles target
+set(CTK_COMPILE_PYTHON_SCRIPT_SKIP_SCRIPT_COPY FALSE)
```

Comment:
Option should also be propagated from the outer build to the inner build, that way configuring CTK specifying `CTK_COMPILE_PYTHON_SCRIPT_SKIP_SCRIPT_COPY` to `TRUE`, will ensure the script specific to CTK are only "compiled" as `.pyc`

Reply:
@jcfr, I've added this option to the top-level CMakeLists.txt. This way, CTK_COMPILE_PYTHON_SCRIPT_SKIP_SCRIPT_COPY should be propagated to the inner build.

Please let me know if I am missing anything.

Reply:
This has been addressed in updated commit by:
- Adding top-level `CTK_COMPILE_PYTHON_SCRIPT_SKIP_SCRIPT_COPY` option
- Calling `mark_as_superbuild`

Additionally, the variable has been added to `CTKConfig.cmake.in`, this will ensure that the behavior of the CMake macro `ctkMacroCompilePythonScript` is consistent across CTK and Slicer.